git-branches

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env tuterm
+
+# tuterm-collection
+#
+# Copyright (c) 2021 The Tuterm Collection Contributors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+depends git
+
+configure() {
+    TUTERM_NAME="git-branches-$(mode)"
+}
+
+run() {
+    git config --global user.name "Tutor"
+    git config --global user.email "tutor@tuterm.io"
+    git config --global init.defaultBranch 'main'
+    m 'Welcome to the git branches tutorial!'
+    m "We are going to learn how to create, checkout, merge and delete branches."
+    echo
+    M "First, we'll create a git repository in the current directory."
+    M 'Type git init to intialize the repo:'
+    c git init
+    M "It's easy to see what branch we are on using the status command:"
+    c git status
+    M "The first line shows us that we are on the main branch."
+    M "Now, we will stage and commit file1 and file2:"
+    c git add file1 file2
```

Comment:
You must create the files before adding them. You can create them in the background by calling `touch file1 file2` or add `c touch file1 file2` to make the user create them. I prefer the latter because it is more transparent - the file's origin won't be mysterious.